torch/csrc/jit/autodiff.cpp

```diff
@@ -213,11 +213,15 @@ class GradientHelper {
  private:
   Node* node;
 
-  SymbolicVariable gradSumToSizeOf(SymbolicVariable v, Symbol input_name) {
+  SymbolicVariable gradSumToSizeOf(
+      SymbolicVariable v,
+      Symbol input_name,
+      SymbolicVariable fw_output) {
     Value* size;
     {
-      WithInsertPoint insert_guard{node};
-      size = SymbolicVariable(node->namedInput(input_name)).size();
+      WithInsertPoint insert_guard{node->next()};
```

Comment:
why does this guard changed from node to node->next()?

Reply:
I added this as a comment, too: We want to use the output of node.